demo/src/Components/BottomSheet/BottomSheet.tsx

```diff
@@ -17,74 +53,148 @@ const BottomSheet = (props: HvComponentProps) => {
     animationDuration: animationDuration
       ? parseInt(animationDuration, 10)
       : 250,
+    contentSections: props.element.getElementsByTagNameNS(
+      namespace,
+      BottomSheetContentSection.localName,
+    ),
     dismissible:
       props.element.getAttributeNS(namespace, 'dismissible') !== 'false',
+    stopPoints: props.element.getElementsByTagNameNS(
+      namespace,
+      BottomSheetStopPoint.localName,
+    ),
+    swipeToClose:
+      props.element.getAttributeNS(namespace, 'swipe-to-close') === 'true',
     toggleEventName: props.element.getAttributeNS(
       namespace,
       'toggle-event-name',
     ),
     visible: props.element.getAttributeNS(namespace, 'visible') === 'true',
   };
+  const hvStyles: HvStyles = {
+    container: Hyperview.createStyleProp(props.element, props.stylesheets, {
+      ...props.options,
+      styleAttr: 'bottom-sheet:container-style',
+    }),
+    handle: Hyperview.createStyleProp(props.element, props.stylesheets, {
+      ...props.options,
+      styleAttr: 'bottom-sheet:handle-style',
+    }),
+    overlay: Hyperview.createStyleProp(props.element, props.stylesheets, {
+      ...props.options,
+      styleAttr: 'bottom-sheet:overlay-style',
+    }),
```

Comment:
prefix is user defined, we should read it from the element
```suggestion
  const hvStyles: HvStyles = {
    container: Hyperview.createStyleProp(props.element, props.stylesheets, {
      ...props.options,
      styleAttr: `${prefix}:bottom-sheet:container-style`,
    }),
    handle: Hyperview.createStyleProp(props.element, props.stylesheets, {
      ...props.options,
      styleAttr: `${prefix}:handle-style`,
    }),
    overlay: Hyperview.createStyleProp(props.element, props.stylesheets, {
      ...props.options,
      styleAttr: `${prefix}:overlay-style`,
    }),
```